pep-0584.rst

```diff
@@ -0,0 +1,73 @@
+PEP: 584
+Title: January 2019 steering council election
+Version: $Revision$
+Last-Modified: $Date$
+Author: Ernest W. Durbin III <ewdurbin@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 3-Jan-2019
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the January
+2019 election for the CPython steering council, as specified in
```

Comment:
Are we officially calling this the CPython steering council, or the Python steering council? I would say it's the latter since that subsumes the former.